src/app.rs

```diff
@@ -177,8 +180,22 @@ pub fn build() -> App<'static, 'static> {
                 .overrides_with("timesort")
                 .overrides_with("sizesort")
                 .overrides_with("extensionsort")
+                .overrides_with("sort")
                 .help("Natural sort of (version) numbers within text"),
         )
+        .arg(
+            Arg::with_name("sort")
+                .long("sort")
+                .multiple(true)
+                .possible_values(&["size", "time", "version", "extension"])
+                .takes_value(true)
+                .value_name("ORDER")
```

Comment:
I think `FIELD` would be a better name here. Or we could use `WORD` which is what gnu ls uses.

Reply:
Whichever you prefer :)

Reply:
I think we will go with `WORD`. Now that I think about it, `version` does not really fit into `FIELD`.